radioactivedecay/__init__.py

```diff
@@ -15,3 +15,4 @@
 from radioactivedecay.decaydata import DecayData, DEFAULTDATA
 from radioactivedecay.radionuclide import Radionuclide
```

Comment:
We'll need to delete this line and remove/replace all references to Radionuclide in the code and tests to Nuclide.